app/assets/javascripts/datepicker.coffee

```diff
@@ -1,3 +1,10 @@
+################################################################################
+# date pickers are used throughout our draft forms (e.g. temporal information) #
+# if you are looking for the UI objects controlling the selection of dates in  #
+# the CMR search related fields (e.g. bulk updates), you should look in the    #
+# datetimepicker files (e.g. datetimepicker.coffee)                            #
+################################################################################
+
```

Comment:
you sure?  search I though was Datepicker, forms (for selecting dates) was Datetimepicker?

Reply:
Yes.
My wild, not based on any real data hypothesis is that at some point we went to make the bulk updates and determined that we wanted a UI that offered maximum specificity for their query (maybe because it was a requirement) and discovered that our datepickers did not do the job.